### PR TITLE
Lightbox memories #21

### PR DESCRIPTION
**Pull request by FlameFractal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -197,7 +197,7 @@ <h3 class="breeze-subtitle">
 				</div>
 				
 				<div class="col-sm-5 col-sm-offset-1">
-                    <iframe width="100%" height="500px" src="https://www.youtube.com/embed/iu-hWvVapbM" frameborder="0" allowfullscreen></iframe>
+                    <iframe height="500px" width="700px" src="https://www.youtube.com/embed/iu-hWvVapbM" frameborder="0" allowfullscreen></iframe>
```
Comment (rhnvrm) [inline index.html]: This does not work for mobile screens. Please restore.

**Comment by rhnvrm** [comment]
@FlameFractal also, there was a hover effect available till this commit https://github.com/snu-breeze/breeze-landing/tree/891279b60b1f988a9b9edc46c0c9182262db20e2 on memories, if you could add that class to the image.

Also, you are already a collaborator through the @snu-breeze/landing team in this repo.

**Comment by FlameFractal** [comment]
Okay. But I don't seem to have the write access to this repo.

File: index.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+    <!DOCTYPE html>
```
Comment (rhnvrm) [inline index.html:1]: Does this need indentation?

Comment (FlameFractal) [inline index.html:1]: oops.

**Pull request merged**